test/lib/ufe/testReorderCmd.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import usdUtils, mayaUtils
+
+import ufe
+import mayaUsd.ufe
+import maya.cmds as cmds
+
+import unittest
+import re
+import os
+
+def findIndex(childItem):
+    hier = ufe.Hierarchy.hierarchy(childItem)
+    childrenList = ufe.Hierarchy.hierarchy(hier.parent()).children();
+    index = childrenList.index(childItem)
+    return index
+
+@unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '2026', 'testReorderCmd is only available in Maya with UFE preview version 0.2.26 and greater')
+class ReorderCmdTestCase(unittest.TestCase):
+    '''Verify the Maya reorder command on a USD scene.'''
+
+    pluginsLoaded = False
+
+    @classmethod
+    def setUpClass(cls):
+        if not cls.pluginsLoaded:
+            cls.pluginsLoaded = mayaUtils.isMayaUsdPluginLoaded()
+
+    @classmethod
+    def tearDownClass(cls):
+        cmds.file(new=True, force=True)
+
+    def setUp(self):
+        # load plugins
+        self.assertTrue(self.pluginsLoaded)
+
+        # open primitives.ma scene in test-samples
+        mayaUtils.openPrimitivesScene()
+
+        # clear selection to start off
+        cmds.select(clear=True)
+
+    def testReorderRelative(self):
+        ''' Reorder (a.k.a move) objects relative to their siblings.
+           For relative moves, both positive and negative numbers may be specified.
+        '''
+        shapeSegment = mayaUtils.createUfePathSegment("|world|Primitives_usd|Primitives_usdShape")
```

Comment:
Because this test won't be executed for UFE v1, it would be much clearer and more maintainable to use UFE path strings:
cylinderPath = ufe.PathString.path('Primitives_usd|Primitives_usdShape,/Xform1/Cylinder1')

Reply:
Thanks. Great point. 

Addressed in 0977cf12e569a8eec7d8b05982d832f112846022